welcome/revhistory_full.adoc

```diff
@@ -4,6 +4,12 @@
 :data-uri:
 :icons:
 :experimental:
+== Wed Apr 27 2016
+.Install Config
+include::install_config/revhistory_install_config.adoc[tag=install_config_wed_apr_27_2016]
+
+.Architecture
+include::architecture/revhistory_architecture.adoc[tag=architecture_wed_apr_27_2016]
```

Comment:
@bfallonf Same comment here about fixing the table titles and putting the intro sentence back in place.